File: app/src/main/java/com/singh/gurtek/kotlinconductor/models/PhotoItem.java
package com.singh.gurtek.kotlinconductor.models;

/**
 * Created by JacksonGenerator on 12/13/17.
 */

import com.fasterxml.jackson.annotation.JsonProperty;


public class PhotoItem {
    @JsonProperty("owner")
    private String owner;
    @JsonProperty("server")
    private String server;
    @JsonProperty("ispublic")
    private Integer ispublic;
    @JsonProperty("isfriend")
    private Integer isfriend;
    @JsonProperty("farm")
    private Integer farm;
    @JsonProperty("id")
    private String id;
    @JsonProperty("secret")
    private String secret;
    @JsonProperty("title")
    private String title;
    @JsonProperty("isfamily")
    private Integer isfamily;
}

File: app/src/main/java/com/singh/gurtek/kotlinconductor/models/PhotoItem.kt
package com.singh.gurtek.kotlinconductor.models

/**
 * Created by Gurtek singh
 * on on 12/13/17
 * gurtek@protonmail.com
 */

import com.fasterxml.jackson.annotation.JsonProperty


data class PhotoItem(
        @JsonProperty("owner")
        private val owner: String? = null,
        @JsonProperty("server")
        private val server: String? = null,
        @JsonProperty("ispublic")
        private val ispublic: Int? = null,
        @JsonProperty("isfriend")
        private val isfriend: Int? = null,
        @JsonProperty("farm")
        private val farm: Int? = null,
        @JsonProperty("id")
        private val id: String? = null,
        @JsonProperty("secret")
        private val secret: String? = null,
        @JsonProperty("title")
        private val title: String? = null,
        @JsonProperty("isfamily")
        private val isfamily: Int? = null
)